src/pages/FlssPrototypes/SampleLoanDisplay.vue

```diff
@@ -0,0 +1,291 @@
+<template>
+	<www-page-minimal
+		:header-theme="headerTheme"
+		:footer-theme="footerTheme"
+	>
+		<div class="simple-campaign-landing">
+			<hr>
+
+			<section
+				class="loan-categories section"
+				id="simpleLoanSection"
+				ref="simpleLoanSection"
+			>
+				<div class="row">
+					<div class="columns">
+						<h2 class="loan-categories__header text-center">
+							Support causes you care about.
+
+							<!--PSD TODO switch to simple dropdown tailored queries-->
+
+							<!--PSD HOWTO pass queries to dev graphql endpoint-->
+							<!-- write a query via appollo here in the page?-->
+
+							<!--PSD HOWTO pass loan ids from query to federation-->
+
+							<!--PSD placeholder page is /pages/FlssPrototypes-->
+						</h2>
+
+						<div class="loan-view-controls">
+							<flss-loan-filters class="loan-view-controls__filters" />
```

Comment:
Prob need to comment out both `flss-loan-filters` and `flss-loan-row` to get the page to load as these components don't yet exist.